server/models/db/student.js

```diff
@@ -10,7 +10,21 @@ const Tables = require('../../constants/tables.json');
 async function getStudent(email, password) {
     return userModel.getUser(email, password, Tables.student, Student);
 }
+// This method inserts a student account given specific information.
+async function insertStudentAccount(first_name, last_name, type, student_id, email, password, phone, school_id) {
```

Comment:
Hi Nancy, 
I would recommend using lowerCamelCase for the variables in the backend and using underscores for the database for greater code consistency. In this case, I would recommend changing the method parameter names to `firstName`, `lastName`, `studentId`, and `schoolId`.

Reply:
Hi Amy,
Thank you for clarifying that!
My action: I've changed the method parameter names to firstName, lastName, studentId, and schoolId.
You can now see these changes in line 14 of server/models/db/student.js

Please confirm if this takes care of your suggestion!

Reply:
Hi Nancy,
Yes, that works. Thank you for making those changes.